TheMovies/TheMovies/Model/MovieModel.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TheMovies.Services;
+using TheMovies.ViewModel;
+
+
+namespace TheMovies.Model
+{
+    public class MovieModel
```

Comment:
You can use the attribute `[JsonProperty("text")]` to map the property to a name in the json. This way you don't need to follow the naming convention of the json in naming your properties.
For example:
`[JsonProperty("poster_path")]`
`public string PosterPath { get; set; }`